src/Goerli-DssSpell.sol

```diff
@@ -45,8 +57,26 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
     function actions() public override {
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralAction
-        // onboardNewCollaterals();
-        // offboardCollaterals();
+        onboardNewCollaterals();
+        drawFromRWA009Urn();
+
+        // Add RWA_TOKEN_FAB to changelog
+        DssExecLib.setChangelogAddress("RWA_TOKEN_FAB", RWA_TOKEN_FAB);
+
+        DssExecLib.setChangelogVersion("1.13.3");
+    }
+
+    function drawFromRWA009Urn() internal {
+        // DSS_PAUSE_PROXY permission on URN
+        RwaUrnLike(RWA009_A_URN).hope(address(this));
```

Comment:
I guess this is part of the set up as in RWA008 we set the operator.

Reply:
Yes that is what I mean. In the case of RWA009, the operator is the pause proxy, so you could set it up in the onboarding (as for RWA008) and not in the minting phase.

Reply:
Looks like you put this in the `onboardRwa008()` section, it should be moved to the `onboardRwa009()` section (tripped me up when I was looking for it).  Lines `165` -> `166`.

Reply:
Yep mentioned it as well in that line